pkg/apis/apis/v1alpha1/types_apibinding.go

```diff
@@ -138,6 +138,13 @@ const (
 
 // APIBindingStatus records which schemas are bound.
 type APIBindingStatus struct {
+	// logicalCluster records the name (not path) of the logical cluster that contains the APIExport.
+	//
+	// +required
```

Comment:
I'm not sure we can make this required - what if the path on the binding has a typo in it and we can't find an actual cluster?

Reply:
Yeah, I wasn't 100% sure on this part. Will make optional.